web/app/css/service-mesh.css

```diff
@@ -24,7 +24,13 @@
   font-size: 16;
   border-radius: calc(var(--base-width)/2);
   font-weight: var(--font-weight-extra-bold);
-  border-radius: var
+  border-radius: var;
```

Comment:
would you mind removing this line? looks like it was incomplete and never removed.

Reply:
Absolutely.